libsolidity/codegen/CompilerUtils.cpp

```diff
@@ -1170,6 +1170,14 @@ void CompilerUtils::popStackSlots(size_t _amount)
 		m_context << Instruction::POP;
 }
 
+void CompilerUtils::popAndJump(unsigned _toHeight, eth::AssemblyItem const& _jumpTo)
+{
+	unsigned amount = m_context.stackHeight() - _toHeight;
```

Comment:
Please either add an assertion that we do not get an overflow here or change the type to `int`.